apps/block_scout_web/assets/css/theme/_dai_variables.scss

```diff
@@ -51,16 +51,6 @@ a.tile-title { color: $secondary !important; }
 $card-background-1: $secondary;
 $card-tab-active: $secondary;
 
-.layout-container {
```

Comment:
Is it necessary to remove it from all other themes (here and below)?

Reply:
Yes, you're right. Somebody had used the `!important` specificity on the banner which was one of the things causing the odd styling in dark mode.  So I removed it and nuked all other styling tied to this banner which wasn't the right call.  I should have only removed from the Celo theme.  Thanks for catching this!